dmutils/content_loader.py

```diff
@@ -83,7 +86,7 @@ def _get_section_filtered_by(self, section, service_data):
         section = section.copy()
 
         filtered_questions = [
-            question for question in section.questions
+            self._add_question_number(question) for question in section.questions
```

Comment:
This isn't needed, the question numbers are added when a new `ContentBuilder` instance is created in [`filter`](https://github.com/alphagov/digitalmarketplace-utils/blob/master/dmutils/content_loader.py#L80).